perfkitbenchmarker/centurylink/util.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for working with Centurylink Cloud resources."""
+
+import os
+import random
+import string
+
+
+def AutoGenerateServerName():
+    """Returns an auto-generated server name"""
+
+    randAlfaKey = ''
```

Comment:
spelling "Alpha" - also to be consistent with the style int the rest of the code base could you please make your local variables use lower case with underscores (e.g. lower_with_under) instead of camel case?